test/index.test.js

```diff
@@ -157,4 +162,33 @@ describe('index test', () => {
               });
         });
     });
+
+    describe('getFile', () => {
+        it('returns error when invalid config object', () => instance.getFile({})
+            .then(() => {
+                assert(false, 'you will never get dis');
+            })
+            .catch(err => {
+                assert.isOk(err, 'Error should be returned');
+                assert.equal(err.name, 'ValidationError');
+            })
+        );
+
+        it('returns not implemented', () => {
+            const config = {
+                scmUrl: 'foo',
+                path: 'testFile',
+                token: 'token'
+            };
+
+            instance.getFile(config)
+              .then(() => {
+                  assert(false, 'you will never get dis');
+              })
+              .catch(err => {
+                  assert.isOk(err, 'Error should be returned');
```

Comment:
Maybe assert.instanceOf(err, Error); I find "isOk" to be a poor definition of an error.